packages/grpc-js/apache-notice.md

```diff
@@ -0,0 +1,4 @@
+The following files contain configuration settings that were derived from [the this commit](https://github.com/google/gts/commit/3b9ab6dd59691f77f5c5c632a44c6762ba4ef7c6) in the Google GTS repository:
```

Comment:
@murgatroid99 is this helpful, necessary, overkill? 🤷

Reply:
I don't know if it's necessary, but I have no objection to including it.